src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Operations/Export/GroupMemberExtractorTests.cs

```diff
@@ -236,6 +263,14 @@ private void SetUpGroupMock()
                         },
                     },
                     new Group.MemberComponent()
+                    {
+                        Entity = new ResourceReference()
+                        {
+                            Reference = _secondPatientReference,
+                        },
+                        Inactive = true,
```

Comment:
Would it be useful to add some resources/tests that also test the condition that checks for the Period Start/End element?